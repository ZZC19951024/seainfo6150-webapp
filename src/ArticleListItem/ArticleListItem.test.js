import React from "react";

describe("ArticleListItem tests", () => {
it("renders correctly", () => {
    const { container } = render(<li/>);
    expect(container).toMatchSnapshot();
  });
});