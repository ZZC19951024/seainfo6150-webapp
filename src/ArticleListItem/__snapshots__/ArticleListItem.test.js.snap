// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ArticleListItem tests renders correctly 1`] = `
<div>
  <li />
</div>
`;
